Gems/Terrain/Code/Source/TerrainSystem/TerrainSystem.cpp

```diff
@@ -253,7 +253,7 @@ AZStd::vector<AZ::Vector3> TerrainSystem::GenerateInputPositionsFromRegion(
 }
 
 AZStd::vector<AZ::Vector3> TerrainSystem::GenerateInputPositionsFromListOfVector2(
-    const AZStd::span<AZ::Vector2>& inPositionsVec2) const
+    const AZStd::span<const AZ::Vector2> inPositionsVec2) const
```

Comment:
no longer passed by reference?

Reply:
Nope. It's a span. So it's just passing a range of iterators. This brings it in line with all other span parameters in this file.

Reply:
@spham-amzn There's no compelling reason to pass a span by reference, it doesn't own data, it just points at data.

Reply:
then there shouldnt be a need for the const anymore
```suggestion
    AZStd::span<const AZ::Vector2> inPositionsVec2) const
```